Tests/mocks/devicesettings.h

```diff
@@ -429,6 +430,74 @@ typedef struct _dsFPDColorConfig_t{
 	dsFPDColor_t color;
 }dsFPDColorConfig_t;
 
+#define MOTION_DETECTION_LEN_MAX                 32
+#define MOTION_DETECTION_SENSITIVITY_IDENTIFIERS 3
+#define SENSITIVITY_MODE_INT                     1
+#define SENSITIVITY_MODE_LEVELS                  2
+#define STR_SENSITIVITY_LOW                      "low"
+#define STR_SENSITIVITY_MEDIUM                   "medium"
+#define STR_SENSITIVITY_HIGH                     "high"
+#define STR_SENSITIVITY_MIN                      "0"
+#define STR_SENSITIVITY_MAX                      "100"
+#define STR_SENSITIVITY_STEP                     "20"
+#define SENSITIVITY_IDENTIFIER_1                 0
+#define SENSITIVITY_IDENTIFIER_2                 1
+#define SENSITIVITY_IDENTIFIER_3                 2
+#define SENSITIVITY_IDENTIFIERS                  3
+
+#define EVENT_TYPE_MOTION_DETECTED               '1'
+#define EVENT_TYPE_MOTION_DISAPEARED             '0'
+
+#define MOTION_DETECTOR                          "FP_MD"
+#define MOTION_DETECTOR_TYPE                     "PID"
+#define MOTION_DETECTION_DESCRIPTION             "The only motion detector"
+#define MOTION_DETECTION_ANGLE                   74
+#define MOTION_DETECTION_DISTANCE                6000
+
+typedef enum _MOTION_DETECTION_Result_t {
+    MOTION_DETECTION_RESULT_SUCCESS,
+    MOTION_DETECTION_RESULT_GENERIC_FAILURE,
+    MOTION_DETECTION_RESULT_INTI_FAILURE,
+    MOTION_DETECTION_RESULT_MODE_ERROR,
+    MOTION_DETECTION_RESULT_INDEX_ERROR,
+    MOTION_DETECTION_RESULT_SENSITIVITY_RANGE_ERROR,
+    MOTION_DETECTION_RESULT_WRONG_INPUT_ERROR,
+    MOTION_DETECTION_RESULT_SET_SENSITIVITY_ERROR,
+} MOTION_DETECTION_Result_t;
+
+typedef enum _MOTION_DETECTION_Mode_t {
+    MOTION_DETECTION_SENSITIVITY_Mode_0, // No Motion Detected
+    MOTION_DETECTION_SENSITIVITY_Mode_1, // Motion Detected
+} MOTION_DETECTION_Mode_t;
+
+typedef struct _MOTION_DETECTION_EventMessage_t {
+    char m_eventType;
+    char m_sensorIndex[MOTION_DETECTION_LEN_MAX];
+} MOTION_DETECTION_EventMessage_t;
+
+typedef struct _MOTION_DETECTION_CurrentSensorSettings_t {
+    unsigned int m_sensitivityMode;
+    char         m_sensorIndex[MOTION_DETECTION_LEN_MAX];
+    char         m_sensorDescription[MOTION_DETECTION_LEN_MAX];
+    char         m_sensorType[MOTION_DETECTION_LEN_MAX];
+    unsigned int m_sensorDistance;
+    unsigned int m_sensorAngle;
+    char         m_sensitivity[MOTION_DETECTION_SENSITIVITY_IDENTIFIERS][MOTION_DETECTION_LEN_MAX];
+
+} MOTION_DETECTION_CurrentSensorSettings_t;
+
+typedef struct _MOTION_DETECTION_Time_t {
+    int m_startTime;
+    int m_endTime;
+} MOTION_DETECTION_Time_t;
+
+typedef struct _MOTION_DETECTION_TimeRange_t {
+    MOTION_DETECTION_Time_t* m_timeRangeArray;
+    int m_nowTime;
+    int m_rangeCount;
+} MOTION_DETECTION_TimeRange_t;
+
+typedef MOTION_DETECTION_Result_t (*MOTION_DETECTION_OnMotionEventCallback)(MOTION_DETECTION_EventMessage_t astEventMessage);
```

Comment:
looks like all those definitions are not from DS (devicesettings) and don't relate to it
please move to a new file e.g. MotionDetection.h